api/src/main/resources/appointment.properties

```diff
@@ -6,9 +6,4 @@ con.setDoInput = true
 
 openmrs.host = localhost
 openmrs.port = 8080
-openmrs.root.uri=http://${openmrs.host}:${openmrs.port}/openmrs/ws/rest/v1
-openmrs.auth.uri=${openmrs.root.uri}/session
-openmrs.user= system-user
-openmrs.password= Admin123
-openmrs.connectionTimeoutInMilliseconds=10000
-openmrs.replyTimeoutInMilliseconds=20000
\ No newline at end of file
+openmrs.root.uri=http://${openmrs.host}:${openmrs.port}/openmrs/ws/rest/v1
```

Comment:
why are the above properties required 
openmrs.host, openmrs.port, openmrs.root.uri 

if these are used to make any API call, then we should change the code. We dont want any loop back API onto the same instance.